easybuild/easyblocks/i/impi.py

```diff
@@ -58,6 +58,12 @@ def extra_options():
         }
         return IntelBase.extra_options(extra_vars)
 
+    def prepare_step(self):
+        if LooseVersion(self.version) >= LooseVersion('2017.2.174'):
+            super(EB_impi, self).prepare_step(requires_runtime_license=False)
```

Comment:
There is says free download at least, same as MKL, this link is maybe a bit more explicit:
https://registrationcenter.intel.com/en/forms/?productid=2558&licensetype=2